packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -2128,6 +2152,51 @@ export interface DiagnosticsCallFeatureState {
     network: NetworkDiagnosticsState;
 }
 
+// @public
+export const Dialpad: (props: DialpadProps) => JSX.Element;
+
+// @public
+export type DialpadMode = 'dtmf' | 'dialer';
+
+// @public
+export interface DialpadProps {
+    dialpadMode?: DialpadMode;
+    disableDtmfPlayback?: boolean;
+    longPressTrigger?: LongPressTrigger;
+    onChange?: (input: string) => void;
+    onClickDialpadButton?: (buttonValue: string, buttonIndex: number) => void;
+    onSendDtmfTone?: (dtmfTone: DtmfTone) => Promise<void>;
+    showDeleteButton?: boolean;
+    // (undocumented)
+    strings?: DialpadStrings;
+    styles?: DialpadStyles;
+    textFieldValue?: string;
+}
+
+// @public
+export interface DialpadStrings {
+    // (undocumented)
+    deleteButtonAriaLabel?: string;
```

Comment:
Adding this comment here to create a thread about the generated screenshots.
Where did the localvideotile go? 
![image](https://github.com/Azure/communication-ui-library/assets/97130533/0444f83f-c67c-456f-9a79-70a3369e21e5)

Reply:
I will rerun them seems weird that they are gone

Reply:
These are consumer snapshots so there shouldn't be a local video tile. But the real question now is how did these pass before.

Reply:
I think know how, its because these are stable snapshots, I bet that we didn't get the tests updated and the new changes to the localTileAPI's removed the local tile as expected